contracts/collectxyz-nft-contract/src/contract.rs

```diff
@@ -91,6 +91,14 @@ pub fn query(deps: Deps, env: Env, msg: QueryMsg) -> StdResult<Binary> {
 
 #[entry_point]
 pub fn migrate(deps: DepsMut, env: Env, msg: MigrateMsg) -> StdResult<Response> {
+    let version = get_contract_version(deps.storage)?;
+    if version.contract != CONTRACT_NAME {
```

Comment:
Assuming this is a new standard?

Reply:
Not sure if it's part of a standard, but it's recommended by that recent CosmWasm [blog post](https://medium.com/cosmwasm/cosmwasm-for-ctos-ii-advanced-usage-ee04ce95d1d0). I think it's worth adding to all our migration handlers as a safeguard to prevent accidentally updating the wrong contract's code when we migrate.